apollo-router/src/executable.rs

```diff
@@ -634,7 +644,7 @@ impl Executable {
 
         let router = RouterHttpServer::builder()
             .configuration(configuration)
-            .and_uplink(opt.uplink_config().ok())
+            .and_uplink(opt.uplink_config(true).ok())
```

Comment:
What's the logic here to decide if we should log a warning or no ?

Reply:
This was just to make sure we only log the warning once. I'll refactor things a bit so we only call the function once and then we don't need the bool.